src/mds/MDSMap.cc

```diff
@@ -462,6 +459,9 @@ void MDSMap::mds_info_t::decode(bufferlist::iterator& bl)
   if (struct_v >= 6) {
     ::decode(standby_for_fscid, bl);
   }
+  if (struct_v >= 7) {
+    ::decode(standby_replay, bl);
+  }
```

Comment:
Okay, so the map will swap it to be a regular standby but its next beacon will request standby-replay and life will then go on as normal? Sounds fine.